osu.Game/Rulesets/Difficulty/Preprocessing/DifficultyHitObject.cs

```diff
@@ -36,6 +41,7 @@ public DifficultyHitObject(HitObject hitObject, HitObject lastObject, double clo
             BaseObject = hitObject;
             LastObject = lastObject;
             DeltaTime = (hitObject.StartTime - lastObject.StartTime) / clockRate;
+            StartTime = hitObject.StartTime / clockRate;
```

Comment:
It's also probably worth exposing `EndTime = hitObject.GetEndTime() / clockRate`.

Reply:
Fixed in 068f00d8a035834b7e21bec15318b6aafd48f9fc